.buildkite/publish/docker-compose.yml

```diff
@@ -126,7 +126,7 @@ services:
       - '3307:3306'
 
   mariadb:
-    image: mariadb:10.9.3
+    image: mariadb:10.7.3
```

Comment:
MariaDB revert. I will be work looking at it soon to understand why it failed on publish, but not in tests.

Reply:
Just checked and the update only bumped the publish pipeline version and not the test pipeline version 🤷🏼 
https://github.com/prisma/prisma/pull/15425/files